feat(ref): sponsor project list (#60)
ulule/developers.ulule.com: content/reference/project.md

| --------- | ------------ |
| `:slug`   | Partner slug |

## List sponsor projects

> Retrieves all the projects of a sponsor with the given user ID.

Reviewer: ```suggestion
Retrieves all the projects sponsored by the sponsor with the given user ID.
```

> 

Reviewer: Specify that the endpoint is paginated
